src/frontends/onnx/frontend/src/op/softmax.cpp

```diff
@@ -9,7 +9,7 @@
 #include "openvino/op/reshape.hpp"
 #include "openvino/op/shape_of.hpp"
 #include "openvino/op/softmax.hpp"
-#include "ov_models/ov_builders/reshape.hpp"
+#include "utils/ov_builders/reshape.hpp"
```

Comment:
@gkrivor do you think we need `ov_builders` in the middle or we can place directly to `utils`? Looks like `utils` already contain so-called builders